masks2d.scad

```diff
@@ -510,6 +535,9 @@ module mask2d_teardrop(r, angle=45, mask_angle=90, excess=0.01, d, anchor=CENTER
 //   anchor = Translate so anchor point is at origin (0,0,0).  See [anchor](attachments.scad#subsection-anchor).  Default: `CENTER`
 //   spin = Rotate this many degrees around the Z axis after anchor.  See [spin](attachments.scad#subsection-spin).  Default: `0`
 //
+// Side Effects:
+//   Sets the default tag to "remove".
```

Comment:
Suggest using more explicit description, like: "Tags the children with "remove" (and hence sets $tag) if no tag is already set."